Prog/NC target tables tabulation.sas

```diff
@@ -0,0 +1,537 @@
+/**************************************************************************
+ Program:  NC target tables tabulation.sas
```

Comment:
this isn't running clean on a batch submit.

Reply:
will move this to issue 7.